### DISCUSSION TITLE
逆時針跑的操場

### DISCUSSION THREAD

**Discussion by forestdew507** [1101-GS4719-C/Python-Community-GS4719-C#2256] [General]
這學期希望能養成運動的習慣，所以前半個學期都 keep on track 的每周到操場跑 2~3 次，一次約 30 min。跑的時候東想西想，突然發現：ㄟ為什麼記憶中校園的操場都設計成逆時針方向跑？

上網查了查原因，覺得滿酷的，分享兩篇網路文章給大家：FYI 於是我在前幾周某次跑的時候也嘗試了久違的順時針跑最外道XD

https://reurl.cc/5Dkzm6
https://reurl.cc/Eg8ex1

尾聲抒發一下～ 要進入冬天了，不管是氣溫還是風速其實都會慢慢變驚人哈哈，好奇入冬後會不會降低你戶外運動的意願？你都怎麼解決！

**Comment by Chattenoire** [comment]
說服自己，運動的好處>窩在宿舍，然後就會拉著自己的身體出去了 (如果沒有可惡的考試@@)

**Comment by ShuliANN** [comment]
天冷確實是會降低戶外運動的意願，但看到自己變得有點鬆弛，還是會逼自己動起來～
而我可能會選擇室內的運動，例如：打羽球、桌球或找溫水的游泳池！

**Comment by yue083139** [comment]
沒想到原來跑操場的方向是有意義的，之前跑步的時候不會特地去注意跑的方向，但今天看到這篇文章後，突然意識到我好像已經習慣逆時針方向慢跑了，之前在家裡附近的公園慢跑也都是下意識的逆時針跑，不知道是習慣成自然還是身體覺得這樣跑比較順呢。之前在冬天的時候真的是人不太想跑步，但是我都會想著不能整天待在家裡，然後用意志力撐完就是了。

**Comment by Mminniehuang** [comment]
因為我非常怕冷，所以天氣一轉涼，我真的完全不會想在戶外運動，對我來說連每天早上出門上課都是煎熬的過程，何況在戶外感受寒冷的氣溫...最近想到體育課就覺得生無可戀...
不過我的朋友就跟我完全不一樣，他反而比較喜歡在冷一點的天氣活動，因為他覺得這樣比較不會流汗，涼爽的天氣也讓人比較有精神！

**Comment by yuan3x2** [comment]
先前都沒注意到這個現象，超酷的！至於運動習慣的部分呢，只能說我是屬於flag從未停止，行動從未開始的類型，因此更別說是寒冷的室外了，如果想要運動又不想受寒的話或許可以考慮去健身房！

**Comment by lai1008** [comment]
金入冬天之後真的每次一出門就會反悔要出去跑步，但為了我的健康著想只能硬著頭皮跑下去，那個風真的是吹到會很不舒服，所以跑完一定要拉筋然後把衣服穿暖活一點

**Comment by Elaine4488** [comment]
天氣冷的時候，確實會讓我更不想去戶外運動。風很大再吸入冷空氣，總會讓人覺得不太舒服。這種時候，我通常會選擇到健身房，使用跑步機或者進行一些肌力訓練。對我來說，這是寒冷天氣裡保持運動習慣的最佳選擇。

**Comment by wcyeee** [comment]
感覺冬天除了很難出門運動，也很容易抽筋或是運動傷害...冬天的溫度和風速有時候真的會一直讓身體涼涼的，所以冬天除了把自己拉出門很辛苦以外，有時候暖身也要做的很足！（然後運動時間就越來越短）

**Comment by kzx-creat** [comment]
上大學之後真的超少在戶外運動(除了學校體育課)，最近想說可以去爬山，但天氣突然變冷，大大降低我的運動慾望。這種天氣運動，總有肢體很僵硬、慢慢上發條的感覺，也很容易喘。

**Comment by 113607514** [comment]
沒想到原來跑操場的方向竟然是有意義的。之前跑步時我從來不特別注意跑步的方向，但今天看到這篇文章後，我才意識到自己似乎已經習慣了逆時針方向慢跑。即使在家附近的公園跑步時，我也是不自覺地選擇逆時針跑，不知道是習慣使然還是身體覺得這樣比較順暢。記得冬天的時候，真的很不想跑步，但我總會告訴自己不能整天待在家裡，於是就靠著意志力堅持跑完。
